workflow/controller/operator.go

```diff
@@ -1669,12 +1669,30 @@ func (woc *wfOperationCtx) markNodePhase(nodeName string, phase wfv1.NodePhase,
 	if node.Completed() && node.FinishedAt.IsZero() {
 		node.FinishedAt = metav1.Time{Time: time.Now().UTC()}
 		woc.log.Infof("node %s finished: %s", node, node.FinishedAt)
-		woc.updated = true
```

Comment:
fat finger?

Reply:
Why would you think so? `node.FinishedAt` is updated.

Reply:
I actually fixed my error, the comment has lingered.